tests/behat/features/ra_grants.feature

```diff
@@ -73,79 +74,79 @@ Feature: A RA(A) should only have access to certain pages
 
   # RA-management create page
   Scenario: An anonymous user can not view the ra-management create page
-    Given I visit the "management/create-ra/00000000-0000-0000-0000-000000000004" page in the RA environment
+    Given I visit the "management/create-ra/00000000-0000-4000-a000-000000000004" page in the RA environment
```

Comment:
Out of curiosity, why did you update the UUID's?

Reply:
Because the first isn't a valid UUID v4.